lib/hanami/view/html_safe_string_buffer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "temple"
+
+# Add temple patch from https://github.com/judofyr/temple/pull/113 (which was since rolled back) to
+# demonstrate the need for the `:capture_generator` setting value to be propogated in order to
+# consistently handle nested captures.
+module Temple
```

Comment:
@timriley Is this still needed, even after https://github.com/judofyr/temple/pull/144 was merged?

Reply:
Nope, this is no longer needed if someone is using the master branch of Temple.

I'm going to keep this patch here so people can test hanami-view even on released versions of temple, but I'll finesse it a bit to remove the method redefinition warning and update the comment to reflect the now-merged PR.

I'll make sure we revisit this and remove the patch before release: https://trello.com/c/bRyiPQEj/38-remove-templegenerator-patch